build.gradle.kts

```diff
@@ -1,14 +1,68 @@
 plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.4"
-  kotlin("plugin.spring") version "1.5.20"
+  kotlin("plugin.spring") version "1.5.21"
+  kotlin("plugin.jpa") version "1.5.21"
 }
 
 configurations {
   testImplementation { exclude(group = "org.junit.vintage") }
 }
```

Comment:
Separating the `test` and `integrationTest` gradle targets based on the folder where tests are located.